src/pradeep/Exam2Prog2.java

```diff
@@ -0,0 +1,42 @@
+package pradeep;
+
+public class Exam2Prog2 {
+	String duplicate = "";
+
+	String findDuplicate(int[] arr) {
+		for (int index = 0; index < arr.length; index++) {
+			for (int index1 = index + 1; index1 < arr.length; index1++) {
+				if (arr[index] == arr[index1]) {
+					duplicate += arr[index];
+				}
+			}
+		}
+		return duplicate;
+	}
+
+	void printSingleOccurance(int[] arr) {
+		duplicate = findDuplicate(arr);
+		String single = "{";
+
+		for (int index = 0; index < arr.length; index++) {
```

Comment:
I would recommend to check below code, small change requires.
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/845/files
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/850/files